### PR TITLE
Update instructions on how to fill PR.

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | I added info about related pull requests and fixing multiple issues.
| Fixed ticket? |

### CODE REVIEW

File: contribute/contribution-guidelines/pull-requests.md [hunk 1]
```diff
@@ -120,14 +120,22 @@ If your code introduces deprecations, please note them here.
 
 ### Fixed ticket
 
-If your Pull Request resolves an existing issue, please note it using the _magic word_ "Fixes", followed by the issue number, like this: `Fixes #12314`.
+If your Pull Request resolves an existing issue, please note it using the _magic word_ "Fixes", followed by the issue number, like this: `Fixes #12314`. If your Pull Request fixes multiple issues, use following syntax: `Resolves #12314, Resolves #56789`.
```
Comment (thomasnares) [inline contribute/contribution-guidelines/pull-requests.md:123]: `Fixes` or `Resolves` ? 😄

**Review by thomasnares — CHANGES_REQUESTED** [review]

File: contribute/contribution-guidelines/pull-requests.md (hunk 1 above)
Comment (Hlavtox) [reply contribute/contribution-guidelines/pull-requests.md:123]: Fixes, but you must use `Resolves` word, if there are multiple issues :D

Comment (thomasnares) [reply contribute/contribution-guidelines/pull-requests.md:123]: You are right, i didn't knew this was specified 👍  (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

**Review by thomasnares — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]
Nice addition :+1:

**Review by nicosomb — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @Hlavtox 

Could you backport the change so it can be available in docs for v8 too?

**Comment by thomasnares** [comment]
Thanks @Hlavtox, i made the backport, you can review it : https://github.com/PrestaShop/docs/pull/1548

**Pull request merged**